### PR TITLE
Use the TEST_SERVER_PORT environment variable in the MonitorFactory

### PR DESCRIPTION
**Pull request by nstapelbroek**
Heya :wave:,

For some reason I was unable to run the PHP webserver used for testing on port 9000 (port was already taken ). After changing the value I discovered that some tests are still stuck because they tried to communicate to the old `9000` port. This PR should create consistency in the location of the testserver and the generated monitor URL's :smile:.

Thanks for the awesome package! :tada:

### CODE REVIEW

**Comment by nstapelbroek** [comment]
Oi, turns out the .styleci had a deprecated flag in it. Removed that :)

File: tests/factories/MonitorFactory.php [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 $factory->define(Monitor::class, function (Faker\Generator $faker) {
     return [
-        'url' => 'http://localhost:9000',
+        'url' => 'http://localhost:'.getenv('TEST_SERVER_PORT'),
```
Comment (freekmurze) [inline tests/factories/MonitorFactory.php]: Could you revert this change? I don't want to use an env var here.

**Comment by nstapelbroek** [comment]
Alright, I'll close this PR since `6a75765` is also included in #144 

On a side note: Would you be interested in a PR that changes port of the testserver into something that isn't used by tools like xdebug? Maybe port `8081`?
